Simplenote/CrashLogging.swift

```diff
@@ -7,19 +7,22 @@ import AutomatticTracks
 /// using `SNCrashLoggingDataProvider`.
 @objc(CrashLogging)
 class CrashLoggingShim: NSObject {
+
+    private static var sharedCrashLogging: CrashLogging?
```

Comment:
Instead of the singleton here, could we instead move this to the `AppDelegate` and explicitly initialize it there?

All of the calls into this shim happen from the AppDelegate anyway, so there they could reference the instance, and we could get rid of all the `static` business going on here?

Reply:
Great suggestion, thank you for the extra context. I'll admit I went with this option simply because it allowed me to keep the change in this file without touching more of the app.

I'll give your suggestion a try, as I'm not happy with singletons either.